catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/sierra/SierraPublishers.scala

```diff
@@ -1,51 +0,0 @@
-package uk.ac.wellcome.platform.transformer.transformers.sierra
-
-import uk.ac.wellcome.models.work.internal.{Organisation, Unidentifiable}
-import uk.ac.wellcome.platform.transformer.source.SierraBibData
-
-trait SierraPublishers {
-
-  // Populate wwork:publishers.
-  //
-  // We use MARC field "260", if populated, otherwise we use field "264".
-  //
-  // In either case:
-  //  - "label" comes from subfield $b
-  //  - "type" is "Organisation".
```

Comment:
@silveroliver We were tying $b as an Organisation originally – according to the new rules, it’s just an Agent. Should we still use the more specific type?